porch/apiserver/pkg/e2e/e2e_test.go

```diff
@@ -706,3 +808,50 @@ func withContent(content configapi.RepositoryContent) repositoryOption {
 		r.Spec.Content = content
 	}
 }
+
+// Creates an empty package draft by initializing an empty package
+func (t *PorchSuite) createPackageDraftF(ctx context.Context, repository, name, revision string) *porchapi.PackageRevision {
+	fullName := fmt.Sprintf("%s:%s:%s", repository, name, revision)
+	pr := &porchapi.PackageRevision{
+		TypeMeta: metav1.TypeMeta{
+			Kind:       "PackageRevision",
+			APIVersion: porchapi.SchemeGroupVersion.String(),
+		},
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      fullName,
+			Namespace: t.namespace,
+		},
+		Spec: porchapi.PackageRevisionSpec{
+			PackageName:    name,
+			Revision:       revision,
+			RepositoryName: repository,
+			Tasks: []porchapi.Task{
+				{
+					Type: porchapi.TaskTypeInit,
+					Init: &porchapi.PackageInitTaskSpec{},
+				},
+			},
+		},
+	}
+	t.CreateF(ctx, pr)
+	return pr
+}
+
+func (t *PorchSuite) mustExist(ctx context.Context, key client.ObjectKey, obj client.Object) {
+	t.GetE(ctx, key, obj)
+	if got, want := obj.GetName(), key.Name; got != want {
```

Comment:
Not sure that this is possible ... presumably obj would be untouched if it was not found?

Reply:
ah yes, you are right. I'll just use GetF